system/config.example.json

```diff
@@ -361,6 +361,9 @@
   // be turned off here. This option has no effect on Blue Burst games - item
   // tracking is always enabled for them.
   "EnableItemTracking": true,
+  
+  // Disable/Enable drops.

```

Comment:
This could do with a bit more explanation.
```
// Enable or disable drops by default in non-BB games (drops are always
// enabled in BB games). The leader can toggle drops in each game with the
// $drop command.
```